assets/subtitles/lJ9C.vtt

```diff
@@ -190,7 +190,7 @@ from important i3 programs
 
 48
 00:02:10.000 --> 00:02:11.000
-like swaybar,
+like sway-bar,
```

Comment:
Actually, the name of the executable is `swaybar`.

Reply:
The screencast showing "man sway-bar" mislead me to think that was the name of the executable. I'm going to fix it in the next commit, thanks.